src/main/java/nextstep/fp/StreamStudy.java

```diff
@@ -27,18 +33,26 @@ public static void printLongestWordTop100() throws IOException {
                 .get("src/main/resources/fp/war-and-peace.txt")), StandardCharsets.UTF_8);
         List<String> words = Arrays.asList(contents.split("[\\P{L}]+"));
 
-        // TODO 이 부분에 구현한다.
+        words.stream().filter(word -> word.length() > TOP_100_MIN_LENGTH)
+                .sorted(Comparator.comparing(String::length))
+                .distinct()
+                .limit(TOP_100_LIMIT_COUNT)
+                .map(String::toLowerCase)
+                .forEach(System.out::println);
```

Comment:
현재는 길이가 짧은 순서로 100개의 단어를 추출하고 있네요!  🤔 
```suggestion
        words.stream().filter(word -> word.length() > TOP_100_MIN_LENGTH)
                .sorted(Comparator.comparing(String::length).reversed())
                .distinct()
                .limit(TOP_100_LIMIT_COUNT)
                .map(String::toLowerCase)
                .forEach(System.out::println);
```

Reply:
아앗...ㅠㅠ 제대로 출력이 되고있는건줄 알았네요...! 감사합니다!